pipelines/README.md

```diff
@@ -137,12 +137,24 @@ oc create -f tekton/test-mlflow-image-pipeline/test-mlflow-image-pipelinerun-ten
 oc create -f tekton/test-mlflow-image-pipeline/test-mlflow-image-pipelinerun-bike-rental.yaml
 ```
 
-Check what pipeline was created and run and what applications started either in OpenShift Console in Pipelines > Pipelines, Topology > select the application and its Resources and Pod logs, or with CLI find the names of the objects with
-
+Check what pipeline was created and run and what applications started either in OpenShift Console in Pipelines > Pipelines,
+or with CLI find the names of the objects with
```

Comment:
```suggestion
Check what pipeline was created and what applications were started either using:
* OpenShift Console in Pipelines > Pipelines
  AND/OR
* CLI; find the names of the objects with:
```

Reply:
```suggestion
Check what pipeline was created and run, and what applications were started either in OpenShift Console in Pipelines > Pipelines,
or using the CLI to find the names of the objects by issuing the following commands
```
@piotrpdev I think your suggestion doesn't render correctly. I think it's ok to remove the `AND/OR` 

![image](https://github.com/opendatahub-io/ai-edge/assets/1930204/900af08d-c388-4cba-9ac7-3ed56180949d)

Reply:
I feel using the bullet points add too much significance to a step that is basically "if you are curious about what happened, these are some hints how to see the results". Or even: "this is what Jan does to make sure he understands what this steps was supposed to achieve".

Plus keeping that command block properly indented under that bullet point is a pain from the editing point of view.

Reply:
> @piotrpdev I think your suggestion doesn't render correctly. I think it's ok to remove the `AND/OR`
> 
> ![image](https://user-images.githubusercontent.com/1930204/273145730-900af08d-c388-4cba-9ac7-3ed56180949d.png)

Fixed, needed spaces 😩

Reply:
> I feel using the bullet points add too much significance to a step that is basically "if you are curious about what happened, these are some hints how to see the results". Or even: "this is what Jan does to make sure he understands what this steps was supposed to achieve".
> 
> Plus keeping that command block properly indented under that bullet point is a pain from the editing point of view.

It's hard to read though 😢 how about this then?

```suggestion
Check what pipeline was created and what applications were started either using the OpenShift Console in Pipelines > Pipelines, or with the CLI; find the names of the objects with:
```

Reply:
I went roughly with the wording suggested by @devguyio.